pkg/cmd/roachtest/tests/backup_fixtures.go

```diff
@@ -132,7 +134,9 @@ func (bd *backupDriver) prepareCluster(ctx context.Context) {
 
 	bd.c.Put(ctx, bd.t.Cockroach(), "./cockroach")
```

Comment:
Instead of putting the burden on the user to make sure their cockroach binary aligns with the version required by the test, couldn't the test stage a released binary?

Reply:
aha! I forgot this is now possible. This simplifies the code nicely. It's a bit goofy that one must always specify a local path to `workload` and `cockroach` even if they are not used, but that seems like a low priority ux papercut.